crypto/bio/bio_lib.c

```diff
@@ -242,6 +242,19 @@ int BIO_method_type(const BIO *b)
     return b->method->type;
 }
 
+int BIO_method_type_final(const BIO *b)
```

Comment:
If we're adding this as a new public API function then it needs to come with some documentation

Reply:
Thanks for reviewing. I'll add the documentation.